### DISCUSSION TITLE
CJK Fonts

### DISCUSSION THREAD

**Discussion by SoNothingMC** [ciderapp/Cider#1238] [General]
*This discussion is a continuation of PR #1209.*

Currently with Noto CJK, Chinese users are not able to view sidebar texts properly without having to zoom into extreme amounts.  
[A quick fix is currently up](https://github.com/SoNothingMC/Cider_RevertCJKFonts), which reverts CJK texts to the system default font.  
But I still want to continue the discussion (and the search of a proper solution), which is why I have created this thread.

Please post suggestions of how we are supposed to move with this going forward.

Thanks in advance!

**Comment by shiinayane** [comment]
Apparently not every Chinese users know why and how to download the theme mentioned,and what they all know is keeping complaining about the awful appearance of CJK fonts.  Theme is not the permanent solution. So I hope that reverting CJK could be merged into main some day. @maikirakiwi
